CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.1.0 (2020-08-12)
+
+- Updated the accessibility rules to ignore invalid autocomplete values to support disabling autofill on Google Chrome. [#41](https://github.com/blackbaud/skyux-sdk-testing/pull/41)
```

Comment:
Are `autocomplete` and `autofill` different? If not, can we stick with one term?

Also, I'm pretty sure this should be pointing to https://github.com/blackbaud/skyux-sdk-testing/pull/40, not https://github.com/blackbaud/skyux-sdk-testing/pull/41.